docs/blog/posts/2024-03-28-colab_gpu.md

```diff
@@ -0,0 +1,236 @@
+---
+date:
```

Comment:
Make sure you add yourself as an author!
https://squidfunk.github.io/mkdocs-material/tutorials/blogs/navigation/#defining-authors

```
  authors:
      - jeffotter
```

Reply:
Done!